src/cli/utils.ts

```diff
@@ -130,7 +131,10 @@ export async function execCommand(
         `"${process.env.SINDRI_FORCE_DOCKER}".`,
     );
   } else if (await checkCommandExists(command)) {
-    logger?.debug(`Executing the "${command}" command locally.`);
+    logger?.debug(
+      { args, command },
```

Comment:
This and below are just making the logging more useful by including the command arguments.